src/buildCommon.ts

```diff
@@ -665,7 +657,12 @@ const retrieveTextFontName = function(
  * - fontName: the "style" parameter to fontMetrics.getCharacterMetrics
  */
 // A map between tex font commands an MathML mathvariant attribute values
-const fontMap: {[string]: {| variant: FontVariant, fontName: string |}} = {
+const fontMap: {
+    [K in string]: {
```

Comment:
I find some of TypeScript's syntax to be a little inconsistent.  For defining type parameters it uses `extends` to describe a type constraint but here it uses `in`. 🤷‍♂️ Not really a big deal, just a little quirk.